src/platform/packages/shared/kbn-esql-utils/src/utils/query_parsing_helpers.ts

```diff
@@ -147,3 +149,29 @@ export const getQueryColumnsFromESQLQuery = (esql: string): string[] => {
 
   return columns.map((column) => column.name);
 };
+
+export const mapVariableToColumn = (
```

Comment:
ℹ️ I have asked the ES team to send us this information on the payload but for now I am using our apis to make this mapping. The adjustments to the api to walk the query and get the params has been merged in a previous PR to make this PR smaller.

Reply:
Clever